recipe/meta.yaml

```diff
@@ -14,21 +14,21 @@ source:
     - avcodec_declfirst.patch  [win and py27]
```

Comment:
@piem it seems that the linter is complaining about this line and not those that you changed.

Reply:
oh! thank you @ocefpaf, i missed that!

this patch has been included upstream now, it needs to be removed.

cheers, piem

Reply:
Done!

Reply:
thanks! that was fast! I also removed the patch itself.